enigma/src/main/java/cuchaz/enigma/source/bytecode/BytecodeSource.java

```diff
@@ -10,13 +10,16 @@
 
 import java.io.PrintWriter;
 import java.io.StringWriter;
+import java.util.List;
 
 public class BytecodeSource implements Source {
 	private final ClassNode classNode;
+	private final List<ClassNode> otherClassNodes;
 	private final EntryRemapper remapper;
 
-	public BytecodeSource(ClassNode classNode, EntryRemapper remapper) {
+	public BytecodeSource(ClassNode classNode, List<ClassNode> otherClassNodes, EntryRemapper remapper) {
```

Comment:
maybe refactor this so it's a clearer why there's a list of "other class nodes"? just looking at this file it would be hard to understand